updating/updating-cluster-cli.adoc

```diff
@@ -42,5 +42,10 @@ include::modules/updating-sno.adoc[leveloffset=+1]
 For information on which machine configuration changes require a reboot, see the note in xref:../architecture/control-plane.adoc#understanding-machine-config-operator_control-plane[Understanding the Machine Config Operator].
 
 include::modules/update-upgrading-cli.adoc[leveloffset=+1]
+include::modules/update-conditional-updates.adoc[leveloffset=+1]
+
+.Additional resources
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Understanding upgrade channels and releases]
```

Comment:
Linked page is "Upgrade channels and release paths", and link ID here is "Understanding upgrade channels and releases." Not sure if matters (inclined to say they should match, though)